README.md

```diff
@@ -1,83 +1,304 @@
-* Before diving into an explicit definition of what a tuple is, it's important for the student to understand the problem they are solving. This can be done by providing a real world example/metaphor they can relate to... or if it's showing them code, it's showing them something they are aware of first.
-
-```swift
-let myFavoriteStarWarsFilms = [
-    "Star Wars - Episode I",
-    "Star Wars - Episode II",
-    "Star Wars - Episode III"
-]
+## Introduction
+You already know that an array is a list of values, and in an array, all of the values have to be of a certain type, i.e. if you have an array of strings, it cannot contain integers.
+
+A tuple is also a list of values, but a tuple's values can be of different types. Also, unlike an array, you can't add values to a tuple once it's defined.
+
+The values in a tuple are generally related to one another. For example, you might have a tuple with a person's name and their age. In this regard, rather than call a tuple a _list_ of values, it would be more accurate to define a tuple as a _group of related values_.
+
+Enough theory, let's dive into some examples!
+
+## Tuples vs Arrays
+
+Did you skip breakfast this morning? Don't worry I have a great assortment of `breakfastBeverages` for you, which should tide you over until your coffee break later in the lesson. :)
+
+We'll start off comparing tuples with arrays. Consider the following array:
+
+```swift
+var breakfastBeverages = ["Milk", "Coffee", "Orange Juice"]
+```
+
+You can add an item to an array, but you can't add anything to a tuple.
+
+```swift
+breakfastBeverages.append("Apple Juice") // ["Milk", "Coffee", "Orange Juice", "Apple Juice"]
+```
+
+An array is a great data structure to use for a list of items of a certain type. Now let's consider a case in which an array could be used, but it wouldn't really be appropriate to do so.
+
+Let's say we wanted to store some properties of a beverage, like whether we prefer a certain beverage hot or cold. We'd need to store the beverage name and either "Hot" or "Cold".
+
+Since both items we want to store are strings, we could technically store them in an array. However what would it be an array of? Beverages? Temperatures?
+
+```swift
+let temperatureArray = ["Coffee", "Hot"]
+let beverageName = temperatureArray[0]
+let beverageTemperature = temperatureArray[1]
+```
+
+We'd really be misusing an array in order to store an item and a _property_ of that item, rather than another item of that type.
+
+Is there a better way? Yes, we can use a tuple! Change those square brackets to parentheses and you've got yourself a tuple.
+
+```swift
+let temperatureTuple = ("Coffee", "Hot")
+```
+
+The tuple above is a group of related values. "Hot" describes the temperature of the "Coffee".
+
+As with an array, we access the values in a tuple by index, however there is a difference in syntax. Tuples use dot syntax, i.e. `.0`, instead of the subscripting syntax that arrays use, i.e. `[0]`.
```

Comment:
Perfect! Again... never thought to correlate the two like this and it was right in front of my face. This is exactly the approach we like to take when describing something new to a student.